### ISSUE TITLE
Parameter to Disable Multiple Root Check

### ISSUE THREAD

**Issue by JKillian** [martinblech/xmltodict#80]
I'm trying to convert a dict to an xml snippet, but this xml snippet is just supposed to be part of a later full document, so it may or may not have one root element. Unfortunately a ValueError is thrown if there is more than one possible root element - it would be great if there was a keyword parameter to disable that check. (Or perhaps when `full_document=False`)

So for example:
```python
xmltodict.unparse({'node': [1,2,3]}, full_document=False)

# would produce something like this without throwing an error:
"""<node>1</node>
<node>2</node>
<node>3</node>
"""
```

**Comment by martinblech** [comment]
Thanks @JKillian for the idea, it will be available in the next release.

**Comment by JKillian** [comment]
Great, thanks for the implementation and quick work @martinblech!

**Comment by JKillian** [comment]
@martinblech Found an edge case actually:

```python
unparse({'node': [1, 2, 3, 4]})
# raises ValueError('document with multiple roots')
```

I think the fix is to pass `full_document` to `_emit` and only do the `if depth == 0 and len(value) > 1:` test if it is a full document.

**Comment by martinblech** [comment]
Thank you for reporting this edge case, @JKillian . It's fixed in 0.9.2 which is already published in PyPI.

**Comment by JKillian** [comment]
Thanks @martinblech!
